qt/widgets/common/src/PropertyHandler.cpp

```diff
@@ -472,16 +470,23 @@ void PropertyHandler::renameChildren(const Mantid::API::CompositeFunction &cf) {
     QString parName = it.key();
     QString fullName = functionPrefix() + "." + parName;
     QtProperty *prop = it.value();
-    Mantid::API::ParameterTie *tie = cf.getTie(cf.parameterIndex(fullName.toStdString()));
+    const auto paramIndex = m_browser->compositeFunction()->parameterIndex(fullName.toStdString());
```

Comment:
Just want to check, `compositeFunction()` can never return a nullptr in this situation?

Reply:
Should always be created in `createCompositeFunction`, called from the `FitPropertyBrowser::init()`